### PR TITLE
Reorganize the structure

### PR DESCRIPTION
**Pull request by ibro45**

### CODE REVIEW

**Comment by surajpaib** [comment]
@ibro45 Can you also update the config file with the `contrib` changes?

**Comment by ibro45** [comment]
Which config file?

**Comment by surajpaib** [comment]
I just realized that the cifar10 doesn't have any contrib imports or custom functionality. Maybe we should create one to demonstrate this?

Or do we want to do it in a separate project?

**Comment by ibro45** [comment]
I think we don't need to demonstrate it right now, we will do that when we have a project that's designed for demonstration purposes

File: lighter/utils/importing.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import sys
```
Comment (surajpaib) [inline lighter/utils/importing.py:1]: Can we rename this file? Naming is inconsistent with the rest.

Maybe `dynamic_imports.py` ?

Comment (ibro45) [reply lighter/utils/importing.py]: Cool, done

**Review by surajpaib — APPROVED** [review]

**Pull request merged**